packages/components/src/Tokens/TokensTextComponents.tsx

```diff
@@ -214,6 +214,14 @@ export const TokenETHFAQQuestion6Text: React.SFC = props => (
   <h3>Why do I need to unlock my Civil tokens before I can sell them?</h3>
 );
 
+export const TokenDonateToCivilFoundationText: React.SFC = () => (
+  <>
+    <h3>Donate to the Civil Foundation</h3>
+    <p>Want to support this project, but don’t want to buy Civil tokens? </p>
+  </>
+);
+
+export const TokenETHFAQQuestion8Text: React.SFC = props => <h3>How do I unlock my tokens?</h3>;
```

Comment:
@sruddy I may have messed up merging with master. Could you confirm this is what we meant to do?

Reply:
Yeah it looks like this is wrong. I think you can just remove `TokenETHFAQQuestion8Text` and you should move your text to the bottom I have everything grouped by component this looks like it's in the middle of the FAQ stuff

Reply:
I removed `TokenETHFAQQuestion8Text`. Thanks!